lib/pbench/server/database/models/models.py

```diff
@@ -0,0 +1,136 @@
+import datetime
```

Comment:
`models.py` is a rather generic name. I don't think we want to define all of our DB models in one file; I put my models in `tracker.py` ... maybe yours should be `users.py` or something like that?